build-farm/platform-specific-configurations/aix.sh

```diff
@@ -51,7 +51,7 @@ if [ "$JAVA_FEATURE_VERSION" -gt 11 ]; then
       fi
     fi
     export JDK_BOOT_DIR="$(eval echo "\$$BOOT_JDK_VARIABLE")"
-    $JDK_BOOT_DIR/bin/java -version | sed 's/^/BOOT JDK: /'
+    $JDK_BOOT_DIR/bin/java -version 2>&1 | sed 's/^/BOOT JDK: /'
```

Comment:
probably a good idea to quote all of the `$JDK_BOOT_DIR/bin/java` like I did in Windows

Reply:
Done.